test/serializers/fieldset_test.rb

```diff
@@ -5,7 +5,7 @@ class Serializer
     class FieldsetTest < ActiveSupport::TestCase
       def test_fieldset_with_hash
         fieldset = ActiveModel::Serializer::Fieldset.new('post' => %w(id title), 'comment' => ['body'])
-        expected = { post: [:id, :title], comment: [:body] }
+        expected = { post: { id: {}, title: {} }, comment: { body: {} } }
```

Comment:
fwiw, all the tests pass.
however, I'm working on something a little more specific to the exact problem, rather than just blanketly using IncludeDirective